docs/Phis_local_installation.md

```diff
@@ -0,0 +1,696 @@
+# Phis Deployment
+
+1. [Prerequisite](#prerequisite)  
+  + [Software](#software)  
+      - [Mongodb + robo3t](#mongodb-robo3t)
+      - [Postgresql + postgis](#postgresql-postgis)
+      - [Netbeans + jdk](#netbeans-jdk)
+      - [Php](#php)
+      - [Apache-tomcat + rdf4j](#apache-tomcat-rdf4j)
+      - [Apache2](#apache2)
+      - [Composer](#composer)
+      - [Git](#git)
+      - [Check install](#check-install)
+  + [Files](#files)
+      - [Web service folder](#web-service-folder)
+      - [Web application folder](#web-application-folder)
+      - [Database and ontologies](#database-and-ontologies)
++ [Installation](#phis-installation)
+    - [Mongodb Database](#mongodb-database)
+    - [Postgresql Database](#postgresql-database)
+    - [Rdf4j ontologies](#rdf4j-ontologies)
+    - [Web service](#web-service)
+    - [Web application](#web-application)
++ [Current errors](#current-errors)
+    - [Problems with netbeans](#problems-with-netbeans)
+    - [Problems with Composer](#problems-with-composer)
+    - [Errors with postgresql](#errors-with-postgresql)
+    - [Errors with the web service](#errors-with-the-web-service)
+    - [Errors with the web application](#errors-with-the-web-application)
+    - [Other problems concerned webapp and web service](#other-problems-concerned-webapp-and-web-service)
+
+## Prerequisite
+### Software
+#### Mongodb + robo3t
+##### Mongodb
+All the information you need to install mongodb correctly is [here](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/#install-mongodb-community-edition/).
+
+Now you have a Mongodb service.
+You can change path to database in the /etc/mongod.conf file. But your location need to exist before starting mongodb service and you have to attribute correct rights on this folder:
+```
+    chown -R mongodb:mongodb <path mongo db>
```

Comment:
Since all the chunks of code you provide work when you're running on linux, you should write it somewhere (some users reading the documentation might run on windows)